File: an2_sem4/MPP/Homework/sapt11/java/repo/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/repo/ISpectacolRepo.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.repo;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;

import java.time.LocalDateTime;

public interface ISpectacolRepo extends IRepo<Integer, Spectacol> {
    Iterable<Spectacol> getBetweenDates(LocalDateTime start, LocalDateTime end) throws EntityRepoException;
}

File: an2_sem4/MPP/Homework/sapt9/cs/FestivalSellpoint/FestivalSellpoint.Repo/ISpectacolRepo.cs
using FestivalSellpoint.Domain;
using System;
using System.Collections.Generic;

namespace FestivalSellpoint.Repo
{
    public interface ISpectacolRepo : IRepo<int, Spectacol>
    {
        IEnumerable<Spectacol> GetBetweenDates(DateTime start, DateTime end);        
    }
}
